feat(container-loader): add captureFullContainerState free function (#27220)
microsoft/FluidFramework: .changeset/wide-foxes-behave.md

---
"@fluidframework/container-loader": minor
"@fluidframework/container-runtime": minor
"__section": feature
---

> Add a new free function `captureFullContainerState` for serializing a container without loading it, for rehydrating into a future session.

Reviewer: ```suggestion
Serialize a container without loading it for rehydrating a future session with `captureFullContainerState`
```
I would mention that it's a free function somewhere in the longer description as opposed to the title.

Author: I'm going to drop the changeset, we don't need to advertise this at the moment.  We can do so when promoting to beta.
